spec/driver_spec.rb

```diff
@@ -2674,8 +2676,11 @@ def which_for(character)
       end
 
       it "can allow specific hosts" do
-        driver.allow_url("example.com")
-        driver.allow_url("www.example.com")
+        configure do |config|
```

Comment:
I was surprised that `configure` works within an `it` block, but after reading through `AppRunner` (specifically, `AppRunner.reset` and `AppRunner.included`, I see how it does. Regardless, this is a bit confusing. However, this is a specific use case within our own specs, so I could be persuaded that it's all right.